jck/jtrunner/JavaTestRunner.java

```diff
@@ -565,6 +565,11 @@ public static boolean jckConfigurationForMultijvmWithNoAgent() throws Exception
 			extraJvmOptions += " -Dfile.encoding=US-ASCII";
 		}
 
+                // testExecutionType of multiJVM_group on Windows causes memory exhaustion, so limit to plain multiJVM
```

Comment:
Since the property is being set in JavaTestRunner, meaning it’d appear in the generated jtb, it will override whatever setting we have for this property in the jti file we use at IBM. If we haven't tested this change on IBM JDKs already, could we pls ensure that this is only done for HotSpot for the moment? We can then do some further testing with IBM before enabling it generally.

Reply:
@Mesbah-Alam Yes, agree.
However, you may not realize the default runtime & compiler.jti files have changed the multiJVM default to "group"
jck.env.testPlatform.multiJVM=Yes_group

https://github.com/adoptium/aqa-tests/commit/44ea825538f9b07398ccd460bfec53d957428203

So you may want me to actually change this to below ?: 
```
                if (getJckVersionInt(jckVersionNo) >= 17 && (platform.contains("win") || isIbmJvm())) {
                        fileContent += "set jck.env.testPlatform.multiJVM \"Yes\";\n";
                }
```

Reply:
@Mesbah-Alam using multiJVM=Yes_group (now the default) causes Windows memory exhaustion in our tests, but vastly increases Jck execution on other platforms.
However, it's probably best for openj9 to use multiJVM=Yes until you have tested with it.

Reply:
Either way, we probably need to apply above modified change or revert https://github.com/adoptium/aqa-tests/commit/44ea825538f9b07398ccd460bfec53d957428203

Reply:
I had not realized earlier that the jti file we use has `jck.env.testPlatform.multiJVM=yes` anyway. So, adding `isIbmJvm()` is not needed. Your original change in this PR should work fine at our end. Please ignore my previous comment. @andrew-m-leonard .